Add `__query_params` kwarg to fragments chaining (#9)
gogrow-dev/prest: README.md

             .post(body: { username: 'juan-apa' })
```

### Using raw/custom/special query parameters

> Because ruby does not accept duplicate keyword arguments on method calls, you can not do the following:

Reviewer: `nit:`
```suggestion
In ruby, duplicate keyword arguments on method calls are not accepted, so you can not do the following:
```

```ruby
# GET https://example.com/api/example?key=1&key2
Prest::Client.new('https://example.com/api').example(key: 1, key: 2).get
```

Because of this and other cases where formatting is very strict/unusual, you can pass a string which will not be formatted to the query parameters. To do this, use the following:

```ruby
# GET https://example.com/api/example?key=1&key2&other=value
Prest::Client.new('https://example.com/api')
             .example(__query_params: 'key=1&key=2', other: 'value')
             .get
```

> The string passed to the keyword argument `__query_params` will not be formatted, and passed as is.

Reviewer: `nit:` it would be nice to put emphasis on this, with `information` or `warning` tags. Check out this: https://github.com/orgs/community/discussions/16925
```suggestion
> Warning
> The string passed to the keyword argument `__query_params` will not be formatted, and passed as is.
```
